frontend/assets/top_containers.bulk.js

```diff
@@ -114,6 +115,18 @@ BulkContainerSearch.prototype.setup_bulk_action_test = function() {
 };
 
 
+BulkContainerSearch.prototype.setup_bulk_action_update_fields = function() {
+  var self = this;
+  var $link = $("#bulkActionUpdateIlsFields", self.$toolbar);
+
+  $link.on("click", function() {
+    AS.openQuickModal("Update ILS Fields", AS.renderTemplate("bulk_action_update_ils_fields", {
```

Comment:
The 'Continue' button output by AS.openQuickModel seems a bit out of place here.  We may need to generate a full dialog template (header, content, footer) so our save/cancel action is in the '<div class="modal-footer">'.